airmozilla/main/static/main/js/calendar.js

```diff
@@ -37,7 +42,10 @@ $(function() {
     startsOnMonday.on('change', function(){
         localStorage.setItem('startsOnMonday', startsOnMonday.prop('checked')? 1 : 0);
         container.fullCalendar('destroy');
+        view = localStorage.getItem('view');
         initCalendar();
+        $('.fc-right button').on('click', save_view);
     });
     initCalendar();
+    $('.fc-right button').on('click', save_view);
```

Comment:
I did a test with that method but the problem with that is there's no way to figure what cause the render event, so for example I can't differentiate between the first render and a render caused by an user who clicked on one of the buttons to change the view.